webserver/templates/data/summary.html

```diff
@@ -0,0 +1,107 @@
+{%- extends 'base.html' -%}
+
+{%- block title -%}
+  Track "{{ metadata['title'] }}" by {{ metadata['artist'] }} - AcousticBrainz
+{%- endblock -%}
+
+{%- block content -%}
+  <div id="page-recording-summary">
+
+    <h2 class="page-title">Track "{{ metadata['title'] }}" by {{ metadata['artist'] }}</h2>
+
+    {%- include 'data/summary-metadata.html' -%}
+
+    {% if submission_count > 1 %}
+      <h3 class="text-center">
+        {% if offset > 0 %}
+          <a href="{{ url_for('data.summary', mbid=metadata['mbid'], n=offset-1) }}"
+             title="View previous submission">&larr;</a>
+        {% endif %}
+        Submission #{{ offset + 1 }} <small>out of {{ submission_count }}</small>
+        {% if offset < submission_count - 1 %}
+          <a href="{{ url_for('data.summary', mbid=metadata['mbid'], n=offset+1) }}"
+             title="View next submission">&rarr;</a>
+        {% endif %}
+      </h3>
+    {% endif %}
+
+    <h3>Low-level information <small>Summary</small></h3>
+
+    <table class="table table-striped table-bordered">
+      <thead><tr><th style="width: 30%;">Tonal & Rhythm</th><th>value</th></tr></thead>
+      <tr>
+        <td><a href="http://essentia.upf.edu/documentation/reference/streaming_ChordsDescriptors.html">key</a></td>
+        <td>{{ data.lowlevel.tonal.key_key }}</td>
+      </tr>
+      <tr>
+        <td><a href="http://essentia.upf.edu/documentation/reference/streaming_ChordsDescriptors.html">scale</a></td>
+        <td>{{ data.lowlevel.tonal.chords_scale }}</td>
+      </tr>
+      <tr>
+        <td><a href="http://essentia.upf.edu/documentation/reference/streaming_Danceability.html">danceability</a></td>
+        <td>{{ data.lowlevel.rhythm.danceability }}</td>
+      </tr>
+      <tr>
+        <td><a href="http://essentia.upf.edu/documentation/reference/streaming_RhythmExtractor2013.html">bpm</a></td>
+        <td>{{ data.lowlevel.rhythm.bpm }}</td></tr>
+      <tr>
+        <td>beat count</td>
+        <td>{{ data.lowlevel.rhythm.beats_count }}</td>
+      </tr>
+    </table>
+
+    {% if data.highlevel %}
+      <h3>High-level information</h3>
+
+      {% macro print_row(row) %}
+        {# Prints <name, value, probability> table row. #}
+          {% set probability = row[2]|float * 100 %}
+          <tr class="{{ 'success' if probability > 75 else (
+                        'warning' if probability > 25 else
```

Comment:
Alright, I'll remove that. Just thought that it would be nice to have more visual indication.